package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "codetour",
-  "version": "0.0.59",
+  "version": "0.0.60",
```

Comment:
Were all of the changes here just the result of running `npm install` or did you have to specifically upgrade anything?

Reply:
result of running npm install, i did not install anything.